### ISSUE TITLE
for dataset & matting result

### ISSUE THREAD

**Issue by guoyt35** [yucornetto/MGMatting#15]
**HI, I use 【com=alpha*img+(1-alpha)*([10, 255,15])】 to get the 'real-world portrait dataset' matting foreground on a green background，but i found the boundary area is not so soft。**
![comp](https://user-images.githubusercontent.com/87117959/124891207-57f70280-e00b-11eb-9ad2-a0da9861b75b.png)
![comp1](https://user-images.githubusercontent.com/87117959/124892117-2894c580-e00c-11eb-992d-e9d40c2832a7.png)
![comp2](https://user-images.githubusercontent.com/87117959/124892162-377b7800-e00c-11eb-8c2c-4c734aa4dbd3.png)
**And I have used your pre-trained model to get my own portrait dataset's matting result, I found that the boundary area performed well, but it missed a lot of inner foreground. My mask is got by using the u^2 net demo.**
![0003](https://user-images.githubusercontent.com/87117959/124892430-74476f00-e00c-11eb-9128-a738b233865f.png)
![0005](https://user-images.githubusercontent.com/87117959/124892535-8aedc600-e00c-11eb-86fd-cfe74eb2b240.png)
![0007](https://user-images.githubusercontent.com/87117959/124892586-94772e00-e00c-11eb-8673-541022d2512a.png)

**Comment by yucornetto** [comment]
Hi,

1. As mentioned in the paper and this github repo, using original image as foreground usually cannot lead to satisfying results. You can try train one using this repo with random alpha blending, which should not be hard to implement, or some other traditional methods (e.g., closed-form matting) to obtain the foreground color. I am occupied recently but will work on releasing the foreground code/model if I get time.

2. The results look weird to me. Could you please confirm that the pretrained weight you are using is MGMatting-RWP-100k instead of MGMatting-DIM-100k?

**Comment by guoyt35** [comment]
I used the MGMatting-RWP-100k pretrained weight to test it again, it performed better a lot in the inner area, thanks a lot!!!

**Comment by hackkhai** [comment]
I tried MGMatting-RWP-100k and still got weird results like he did :(
